Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-
-# combat
+# Combat
 
 evolutionmenu-combat-armblade-name = Arm Blade
 evolutionmenu-combat-armblade-desc =
-    Reform one of your arms into a grotesque blade, composed of bone and flesh, able to pry open airlocks and cut through your foes like butter.
-    Costs 15 chemicals.
+    We recreate one of our arms into a grotesque blade, capable of prying open airlocks and maiming our enemies. Cannot be used in our lesser form.
+    Costs a small amount of chemicals.
 
 evolutionmenu-combat-boneshard-name = Bone Shard
 evolutionmenu-combat-boneshard-desc =
-    Break off shards of your bone and shape them into a throwing star which embeds into your foes. But a one timer opportinuty.
-    Costs 15 chemicals.
+    We break off shards of sharp bone to use as projectile weapons. Cannot be used in our lesser form.
```

Comment:
can drop "weapons" from first sentence. We break off shards of sharp bone to use as projectiles.